security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,29 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-0-0"]
+== OpenShift Compliance Operator 1.0.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.0.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:1682[RHBA-2023:1682 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-1-0-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The Compliance Operator is now stable and the release channel is upgraded to `stable`. Future releases will follow link:https://semver.org/[Semantic Versioning]. To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
+
+[id="compliance-operator-1-0-0-bug-fixes"]
+=== Bug fixes
+
+* Before this update, the `rhcos4-ensure-logrotate-activated` rule failed after auto-remediation was applied in Red Hat Enterprise Linux (RHEL) 9 environments. With this update, rules with auto-remediation available have a `PASS` result after the auto-remedations are applied. (link:https://issues.redhat.com/browse/OCPBUGS-10769[*OCPBUGS-10769*])
+
+* Before this update, the `compliance_operator_compliance_scan_error_total` metric had an `ERROR` label with a different value for each error message. With this update, the `rhcos4-ensure-logrotate-activated` rule does not increase in values. (link:https://issues.redhat.com/browse/OCPBUGS-1803[*OCPBUGS-1803*])
```

Comment:
That rule has nothing to do with that metric

Reply:
Something like "with this update, the error message has been removed from the metric, thus decreasing the cardinality of the metric in line with best practices"